README.md

```diff
@@ -195,6 +195,7 @@ The following parameters are specific to each Codacy component.
 | `codacy-api.metrics.serviceMonitor.enabled`         | Create the ServiceMonitor resource type to be read by prometheus operator.                          | `false`                                      |
 | `codacy-api.metrics.grafana_dashboards.enabled`     | Create the ConfigMap with the dashboard of this component. Can be imported through grafana sidecar. | `false`                                      |
 | `worker-manager.config.workers.dedicatedMax`        | TBD                                                                                                 | `2`                                          |
+| `worker-manager.config.workers.resolveDbsIps`       | Resolve Database's ips at workers start-up (to avoid DNS issues)                                    | `false`                                      |
```

Comment:
From the standpoint of a layman, the way the description is written makes it seem that `true` is actually the option that resolves the hostname to an IP address every time the workers start, as one would expect.

It was only by reading the description of this issue [CY-1309](https://codacy.atlassian.net/browse/CY-1309) that I understood why it works exactly the opposite way. :slightly_smiling_face: 

I think that unless someone knows the internal architecture of these components it's hard to explain how this setting works and what is the impact of having it set to either `true` or `false`.

Reply:
👍 

Ok so I'll close this PR, do the change in the worker manager configuration instead and not expose this on the chart